src/main/resources/templates/login.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
This is purely optional. This is the login page, there's also a sign up link at the end of the form. The styling can be better but its a good start.